swagger_templates/api.service.mustache

```diff
@@ -0,0 +1,47 @@
+{{#operations}}
+/// <reference path="../interfaces.ts" />
+
+/* tslint:disable:max-line-length */
+/* tslint:disable:eofline */
+var resourceBase = <Resource>{
+  "name": "{{toSnakeCase pathPrefix}}",
+  "comment": "",
+  "properties":[],
+  "actions": [
+  {{#operation}}{{^formParams}}
+  {{#contents}}
+  {
+    "name": "{{operationId}}",
+    "method": "{{toUpperCase httpMethod}}",
+    "collection": {{#queryParams}}{{#eq baseName "limit"}}true||{{/eq}}{{/queryParams}}false,
```

Comment:
In order to figure out if an operation supports pagination I use the following logic. It will default to false but if one of the query params is `limit` then this route supports pagination. In order to support these two cases I appended true with `||` so you will see that operations that support pagination will have a collection value of `true || false`